pkgs/development/libraries/wxwidgets/wxGTK32.nix

```diff
@@ -0,0 +1,173 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchurl,
+  gnome2,
```

Comment:
```suggestion
{ lib
, stdenv
, fetchFromGitHub
, fetchurl
, gnome2
```
Formatting using alejandra is not allowed.

Reply:
I now reformated using nixpkgs-fmt.